worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -244,14 +251,16 @@ public void formats(Actor actor) throws WorldEditException {
     @Command(
             aliases = {"list", "all", "ls"},
             desc = "List saved schematics",
-            max = 0,
-            flags = "dn",
+            min = 0,
+            max = 1,
+            flags = "dnp",
             help = "List all schematics in the schematics directory\n" +
                     " -d sorts by date, oldest first\n" +
-                    " -n sorts by date, newest first\n"
+                    " -n sorts by date, newest first\n" +
+                    " -p <page> prints the requested page\n"
     )
     @CommandPermissions("worldedit.schematic.list")
-    public void list(Actor actor, CommandContext args) throws WorldEditException {
+    public void list(Actor actor, CommandContext args, @Switch('p') @Optional("1") int page) throws WorldEditException {
```

Comment:
hmm does that @Optional("1") mean "1 optional argument to -p" or does it mean "-p is optional and requires one argument".
In the first case, your code doesn't match your help blurb
In the second case, @Optional is veeeery strange...